packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -800,6 +800,17 @@ export default class SalesforceCRMService implements CRM {
     const conn = await this.conn;
     const emailDomain = email.split("@")[1];
 
+    // First check if an account has the same website as the email domain of the attendee
+    const accountQuery = await conn.query(
+      `SELECT Id, Website FROM Account WHERE Website LIKE '%${emailDomain}%'`
```

Comment:
We're doing a wild card search here where the website just has to contain the email domain. This would cases where the domain is "test.com" but the website could be stored as "www.test.com"